heatpump.yaml

```diff
@@ -1070,119 +1088,119 @@ sensor:
       return t1s_dhw;
 
 binary_sensor:
-  # Register: 0
+  # Register: 0 ->Is read as sensor from modbus to be fully switch functiuoning
```

Comment:
I want to keep the "reserved" bits, which makes the extra comment not needed. Can you revert it.

Reply:
Ok. Like you wish. I've commented reserved to leave original code if in future manufacturer extends registers to faster adopt, but for me leaving these is waste of time, memory and makes extra data transfer to homeassistant or other dashboards and makes more garbage on integration with parts which potentially on 99% never be used.